### PR TITLE
Add contribution guide for review.

### PR DESCRIPTION
**Pull request by eclectic-coding**
I have edited a contribution guide for review. Look over carefully and let me know if changes need to be made. 

In addition, we probably need to reference this completed guide in the README.

This PR addressed #125

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+## Forking the Project
+
+### Setting Up Your System
+
+1.  Install [Git](https://git-scm.com/) or your favorite Git client.
+2.  (Optional) [Setup an SSH Key](https://help.github.com/articles/generating-an-ssh-key/) for GitHub.
+
+### Forking Developer Portfolios
+
+1.  Go to the top level page of this [repository](https://github.com/polishedwp/modulus)
```
Comment (Easybuoy) [inline CONTRIBUTING.md]: I'm guessing this line is meant to be `https://github.com/emmawedekind/developer-portfolios`

**Comment by geekskick** [comment]
I'm not sure the focus of the guide is correct: There is a link at the top for a guide to creating a pull request, then in-depth septs to doing it for most of the guide. There is no mention of keeping the list in alphabetical order, for example. It's a fantastic guide for how to make a PR and use git as a tool, but since this is aimed at developers I'm not sure it's the right place for it?

**Comment by eclectic-coding** [comment]
@geekskick Thank you for looking at this document. The purpose for this document, is to set procedures on how to interact with the repository in an efficient way. Normally, contribution documents also evolves to address, problems with PR's that have been submitted, and to address reoccurring problems. It lays out these procedures and helps new developers with limited GitHub experience.

FYI. If you look in the **Creating a Pull Request** section (item number 8), the directions include the alphabetically listing of new entries.

File: CONTRIBUTING.md (hunk 1 above)
Comment (eclectic-coding) [reply CONTRIBUTING.md]: That is correct, I will go back through and make sure the links and references are correct.

**Comment by geekskick** [comment]
Ah I hadn't caught that part enough - apologies. 

I still believe that it's too in-depth on the git usage front. If I was adding a link, as a git user (which I suspect most are as it's GitHub!) then there's too much low level detail in there for the salient points to be clear. My personal opinion is that the link to the tutorial at the top is enough. 

That doesn't take away that the content is really good though, just perhaps too in-depth for me personally. Thanks for engaging in discussion about it :)

**Comment by eclectic-coding** [comment]
@geekskick Thank you for your response and I do understand you position. However, it was a document like this one, that helped me when I made my first Pull Request a few years ago. It helped me with the fear of inexperience, by walking me through the process. Maybe it will help others.

**Comment by gep13** [comment]
@eclectic-coding @geekskick can I suggest that we have a [TL/DR](https://www.dictionary.com/e/acronyms/tldr/) section at the top?  i.e. create a PR with new addition in alphabetical order, and then the remainder of the document is left in tact with anyone not familiar with the process.

**Comment by eclectic-coding** [comment]
@gep13 I think that is a great idea. I will make the revisions after Christmas. 

Does someone want to add an introduction sentence or two to the README about this document? Or would you like me to do that?

**Pull request merged**